packages/graphql/src/index.ts

```diff
@@ -40,3 +43,7 @@ export {
     Neo4jGraphQLAuthenticationError,
     Neo4jGraphQLForbiddenError,
 } from "./classes";
+
+export * as directives from "./graphql/directives";
+
```

Comment:
Are the `scalars` also something we want to export?
Not sure about the `enums`, could they even be used for for instance autocompletion in the Toolbox?